Automattic/wp-calypso
# A4A: Sites dashboard - fixing site sort functionality

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/163

## Proposed Changes

* This PR fixes the sorting functionality on the A4A sites dashboard.
* It does this by coping over the SiteSort component to A4A and renaming it, then conditionally loading it in the Dataviews component if we are in A4A. The reason it has been copied is because the context in A4A is different than that used in Jetpack Manage.

What doesn't work yet:
* Clicking the preview pane resets the sort, so when the sort is desc it resets it to asc (default) first, meaning the preview pane won't open. Issue for that (for sort, search and filters) is here: https://github.com/Automattic/automattic-for-agencies-dev/issues/164

**Update:** With the latest commits, this part is now working

## Testing Instructions

For the below testing steps, make sure to run `yarn start-a8c-for-agencies` locally and then visit `http://agencies.localhost:3000/sites`.

* Click on the site heading in the sites table (on the arrows) to enable sort. Each click should go between enabling sort, then asc and desc.
* Save a URL with sort fields in it, and manually enter it in a new tab. It should work and retain the correct sort direction.

## Pre-merge Checklist
